Phase02/Phase02-FullTextSearch/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Phase02_FullTextSearch.Services;
+using Phase02_FullTextSearch.UI;
+
+namespace Phase02_FullTextSearch;
+
+internal class Program
```

Comment:
Use explicit "private" modifier

Reply:
We can't make the top level classes in a namespace private, but if the purpose was to change the Main method access modifier, I applied it.

Reply:
Yeah I meant Main method